ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -17,7 +17,11 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 // See the feature message definitions for descriptions.
 message ExperimentalFeatures {
   ExperimentalSelfServiceErrorCodes self_service_error_codes = 1;
+  ExperimentalStaticTime static_time = 2;
 }
 
 // GRPC self-service error codes are returned by the Ledger API.
 message ExperimentalSelfServiceErrorCodes {}
+
+// Ledger is in the static time mode and exposes a time service.
+message ExperimentalStaticTime {}
```

Comment:
Why experimental?

Reply:
I have followed @meiersi-da suggestion as supporting the static-time (clandestinely as it may) and releasing it as an official feature is two different things.

Reply:
ok